openhumans/models.py

```
@@ -95,6 +95,19 @@ def _refresh_tokens(self, client_id, client_secret):
             self.token_expires = self.get_expiration(data['expires_in'])
             self.save()
 
+    def delete_single_file(self, file_id, file_basename):
```

`file_id` and `file_basename` should both be optional -- at least one of the two should be provided, but not both. You can accomplish by setting to default values of None. Then check that one (but not both) are not None in the function.

I _think_ ohapi should have an error if both are provided. (I'm curious if that's true.)

---

It seems like this function is still requiring both parameters?

I apologize for lack of clarity, but I think this function should accept one and only one of the two parameters. i.e. `file_id` OR `file_basename`, but not both, and not neither. I believe both parameters need to be optional in the function definition, and it should raise an error if it receives neither, or both.